config/main.py

```diff
@@ -952,6 +964,29 @@ def neighbor(ipaddr_or_hostname, verbose):
     """Start up BGP session by neighbor IP address or hostname"""
     _change_bgp_session_status(ipaddr_or_hostname, 'up', verbose)
 
+#
+# 'remove' subgroup ('config bgp remove ...')
+#
+
+@bgp.group()
+def remove():
+    "Remove BGP neighbor configuration from the device"
+    pass
+
+@remove.command('neighbor_ip')
+@click.argument('neighbor_ip', metavar='<neighbor_ip>', required=True)
+def remove_neighbor_with_ip(neighbor_ip):
```

Comment:
Do you prefer to have separate commands for neighbor IP and neighbor name?

For shutting down/starting up BGP sessions (`config bgp shutdown neighbor ...` / `config bgp startup neighbor ...`), I allow the user to enter either the neighbor's IP address *or* hostname without the `neighbor_ip` or `neighbor_name` qualifiers. You can see how I did it here: https://github.com/Azure/sonic-utilities/blob/a134fd13a80415ce41f7afbe85d571aa9824f615/config/main.py#L941

Would you rather match the same approach here?

Reply:
Thanks @jleveque.
submitted another iteration with single command to accommodate either ip address or hostname